### PR TITLE
cleans up atmos defines

### PR DESCRIPTION
**Pull request by duncathan**
a step forward in my quest of documenting atmos

### CODE REVIEW

File: code/__DEFINES/atmospherics.dm [hunk 1]
```diff
@@ -107,65 +112,71 @@
...
-
-// used for device_type vars; used by DEVICE_TYPE_LOOP
+//used for device_type vars; used by DEVICE_TYPE_LOOP
 #define UNARY		1
 #define BINARY 		2
 #define TRINARY		3
 #define QUATERNARY	4
 
-// this is the standard for loop used by all sorts of atmos machinery procs
+//TODO: finally remove this bullshit
```
Comment (duncathan) [inline code/__DEFINES/atmospherics.dm]: .

**Comment by praisenarsie** [comment]
documenting as in documenting or some dumb comments in all atmos-related code

also will this documentation include assmos?

**Comment by duncathan** [comment]
a quick look at https://github.com/tgstation/tgstation/blob/master/code/modules/atmospherics/gasmixtures/gas_types.dm and https://github.com/tgstation/tgstation/blob/master/code/modules/atmospherics/gasmixtures/reactions.dm should tell you almost everything you need to know about assmos; it's very straightforward and gas mixtures overall are already very well documented. I'm not sure what you're asking for here

File: code/__DEFINES/atmospherics.dm (hunk 1 above)
Comment (Cyberboss) [reply code/__DEFINES/atmospherics.dm]: finally

**Pull request merged**